### PR TITLE
Translated in English

### PR DESCRIPTION
**Pull request by AndraB90**

### CODE REVIEW

File: src/example1/Main.java [hunk 1]
```diff
@@ -5,39 +5,30 @@
...
-            } catch (InterruptedException e) {
 
-            }
-        }
-
-         */
-        // Afiseaza cate Thread-uri sunt active in acest moment
-        System.out.println("Numarul thread-urilor active: "+Thread.activeCount());
+        // Print how many Threads are active now
+        System.out.println("Number of active Threads: "+Thread.activeCount());
```
Comment (GabrielCorlan) [inline src/example1/Main.java]: please format the code style using command + alt + L

File: src/example1/Main.java [hunk 2]
```diff
@@ -5,39 +5,30 @@
...
-        //Schimba prioritatea Thread-uliu principal
+        // Change priority of main Thread
         Thread.currentThread().setPriority(10);
-        System.out.println("Prioritatea Thread-ului "+Thread.currentThread().getName()+" este: "+Thread.currentThread().getPriority());
+        System.out.println("Priority of Thread "+Thread.currentThread().getName()+" is: "+Thread.currentThread().getPriority());
 
-        // Afiseaza daca Thread-ul current este activ
-        System.out.println("Thread-ul "+Thread.currentThread().getName()+" este activ: "+Thread.currentThread().isAlive());
+        // Show if current Thread is active
+        System.out.println("Thread "+Thread.currentThread().getName()+" is activ: "+Thread.currentThread().isAlive());
```
Comment (GabrielCorlan) [inline src/example1/Main.java]: Unnecessary empty spaces

File: src/example1/Main.java (hunk 1 above)
Comment (AndraB90) [reply src/example1/Main.java]: Resolved

File: src/example1/Main.java (hunk 2 above)
Comment (AndraB90) [reply src/example1/Main.java]: Resolved :)

**Review by GabrielCorlan — APPROVED** [review]
Looks good! Approved.

**Pull request merged**